tmt/steps/discover/fmf.py

```diff
@@ -148,6 +181,26 @@ def go(self):
         if names:
             self.info('names', fmf.utils.listed(names), 'green')
 
+        only_modified = self.get('only_modified')
+        ref_url = self.get('reference_url')
+        if ref_url:
+            self.info('reference_url', ref_url, 'green')
+            self.debug(f"Fetch also '{ref_url}' as 'reference'.")
+            self.run(['git', 'remote', 'add', 'reference', ref_url],
+                     cwd=testdir, shell=False)
+            self.run(['git', 'fetch', 'reference'], cwd=testdir, shell=False)
+        if only_modified:
+            reference_ref = self.get('reference_ref')
+            self.info('reference_ref', reference_ref, 'green')
+            output = self.run(['git', 'log', '--format=', '--stat',
+                               '--name-only', f"{reference_ref}..HEAD"],
+                              cwd=testdir, shell=False)[0]
+            modified = set(f"^/{re.escape(name)}$"
+                           for name in map(os.path.dirname, output.split('\n'))
+                           if name)
+            self.debug(f"Limit to modified test dirs: {modified}")
```

Comment:
```suggestion
            self.debug(f"Limit to modified test dirs: {modified}", level=3)
```
Let's move this to the third debug level.

Reply:
OK, will change in the next revision.